draft-ietf-anima-brski-prm.md

```diff
@@ -858,137 +860,148 @@ The following sub sections split the interactions shown in {{exchangesfig_uc2_al
+Optionally, TLS MAY be used to provide privacy for this exchange between the Registrar-Agent and the pledge, see {{pledgehttps}}.
 
-{{exchangesfig_uc2_1}} shows the Pledge Voucher-Request aquisition and the following subsections describe the corresponding artifacts. 
+{{exchangesfig_uc2_1}} shows the acquisition of the Pledge Voucher-Request (PVR) and the following subsections describe the corresponding artifacts. 
 
 ~~~~ aasvg
-+--------+   +-----------+   +-----------+    +--------+  +---------+
-| Pledge |   | Registrar-|   | Domain    |    | Domain |  | MASA    |
-|        |   | Agent     |   | Registrar |    | CA     |  |         |
-+--------+   +-----------+   +-----------+    +--------+  +---------+
-   |                |                 |              |   Internet |
-   ~                ~                 ~              ~            ~
-   (1) Trigger Pledge Voucher-Request
-   ~                ~                 ~              ~            ~
-   |                |                 |              |            |
-   |<-- opt. TLS -->|                 |              |            |
-   |<---- tPVR -----|                 |              |            |
-   |----- PVR ----->|                 |              |            |
-   |                |                 |              |            |
-   ~                ~                 ~              ~            ~
++--------+    +------------+    +-----------+    +--------+    +------+
+| Pledge |    | Registrar- |    |  Domain   |    | Domain |    | MASA |
+|        |    |   Agent    |    | Registrar |    |   CA   |    |      |
++--------+    +------------+    +-----------+    +--------+    +------+
+ |                  |                 |                 |   Internet |
+ ~                  ~                 ~                 ~            ~
+(1) Trigger Pledge Voucher-Request
+ ~                  ~                 ~                 ~            ~
+ |                  |                 |                 |            |
+ |<----opt. TLS---->|                 |                 |            |
+ |<------tPVR-------|                 |                 |            |
+ |--------PVR------>|                 |                 |            |
+ |                  |                 |                 |            |
+ ~                  ~                 ~                 ~            ~
 ~~~~
-{: #exchangesfig_uc2_1 title='PVR aquisition exchanges' artwork-align="left"}
+{: #exchangesfig_uc2_1 title="PVR acquisition exchange" artwork-align="center"}
 
-### Request Artifact: Pledge Voucher-Request Trigger (tPVR)
-
-Triggering the pledge to create the PVR is done using HTTP POST on the defined pledge endpoint: "/.well-known/brski/tpvr"
-
-The Registrar-Agent PVR trigger Content-Type header is: `application/json`.
-Following parameters are provided in the JSON object:
+The Registrar-Agent triggers the pledge to create the PVR via HTTP POST on the well-known pledge endpoint `/.well-known/brski/tpvr`.
```

Comment:
Since you state yourself that you want transport independence, the definitions should also be split into transport aspects (here HTTP request and response) and artifact aspects.

I started to move the transport aspects into the "main subsection" under the figure and only have the artifact aspects in the subsubsections. Maybe we should not call it "Request Artifact" and "Response Artifact", as that only fits a request-response transport...